policy/release/cve.rego

```diff
@@ -66,7 +66,7 @@ warn contains result if {
 # METADATA
 # title: Deprecated CVE result name
 # description: >-
-#   The `CLAIR_SCAN_RESULT` result name has been deprecated, and has been
+#   The `SCAN_OUTPUT` result name has been deprecated, and has been
```

Comment:
I think we'd still want to keep this change as well, as it would help the EC policies to catch if any of the outdated Clair tasks are still using the CLAIR_SCAN_RESULT result name.

Reply:
+1